lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -232,6 +235,12 @@ type GetUTxOsStatistics = "wallets"
     :> "utxos"
     :> Get '[JSON] ApiUtxoStatistics
 
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/forceResync
+type ForceResyncWallet = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "utxos"
```

Comment:
why this? :thinking:

Reply:
you mean why `utxos` specifically or, why an extra sub-level ?

Reply:
yeah, utxos...more thoughts on this below